lib/chef-dk/cli.rb

```diff
@@ -84,13 +86,26 @@ def run
     def handle_options
       parse_options(argv)
       if config[:version]
-        msg("Chef Development Kit Version: #{ChefDK::VERSION}")
+        show_version
       else
         show_help
       end
       exit 0
     end
 
+    def show_version
+      msg("Chef Development Kit Version: #{ChefDK::VERSION}")
+
+      ["chef-client", "berks", "kitchen"].each do |component|
+        result = Bundler.with_clean_env { shell_out("#{component} --version") }
+        if result.exitstatus != 0
+          next
```

Comment:
We could print out "version: unable to determine" or "version: error please see log" or just print something to debug. What do you guys think?